cmd/kube-apiserver/app/testing/testserver.go

```diff
@@ -158,7 +158,7 @@ func StartTestServer(t Logger, instanceOptions *TestServerInstanceOptions, custo
 	if err != nil {
 		return result, fmt.Errorf("failed to create a client: %v", err)
 	}
-	err = wait.Poll(100*time.Millisecond, 30*time.Second, func() (bool, error) {
+	err = wait.Poll(100*time.Millisecond, 60*time.Second, func() (bool, error) {
```

Comment:
doesnt seems related w/ the goal of this pull, am i missing sth?

Reply:
this is increasing the timeout for the test server to come up in the integration test, which is necessary because the post start hook takes 30 seconds to time out, and the test previously also timed out after 30 seconds. by increasing the time out of the test, it will allow it to wait long enough for the post start hook to time out